AboutMe.Web/Controllers/AccountController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Application.Dtos.Identity;
+using Core.Constants;
+using Core.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AboutMe.Web.Controllers
+{
+    [AllowAnonymous]
+    public class AccountController : Controller
+    {
+        private readonly SignInManager<User> _signInManager;
+        private readonly UserManager<User> _userManager;
+
+        public AccountController(
+            UserManager<User> userManager,
+            SignInManager<User> signInManager)
+        {
+            _userManager = userManager;
+            _signInManager = signInManager;
+        }
+
+        [HttpGet]
```

Comment:
these methods are not routed properly, add `[Route]` attributes or add the desired route to the method attributes. Otherwise the runtime will not be able to route your requests correctly